gapipy/resources/base.py

```diff
@@ -55,6 +55,9 @@ def __repr__(self):
         else:
             return '<{}: {}>'.format(self.__class__.__name__, self.id)
 
+    def __hash__(self):
+        return hash(self.__repr__())
```

Comment:
@siamalekpour Ah, yes, it will *begin* life as a stub only containing the resource name and its `id`. If you try to [access any of its other attributes](https://github.com/gadventures/gapipy/blob/4f9a27daf913c20f1a16959e8532134f10f56c04/gapipy/resources/base.py#L42) it will hit the API to fetch its missing data and set `self.is_stub = False` (see also the [`fetch` method](https://github.com/gadventures/gapipy/blob/4f9a27daf913c20f1a16959e8532134f10f56c04/gapipy/resources/base.py#L23)).

(`fetch` may just wind up hitting the cache if the resource is available rather than going to the API -- it winds up calling [`Query.get`](https://github.com/gadventures/gapipy/blob/45d3dee79e47dddc27934d626dc9d66ad2483f14/gapipy/query.py#L37) if you're curious about that...)